src/Simplex/Messaging/Agent/Store/SQLite/Schema.hs

```diff
@@ -57,10 +57,10 @@ sndQueues =
       encrypt_key BLOB NOT NULL,
       sign_key BLOB NOT NULL,
       status TEXT NOT NULL,
-      PRIMARY KEY(host, port, snd_id),
-      FOREIGN KEY(host, port) REFERENCES servers(host, port),
-      FOREIGN KEY(conn_alias)
-        REFERENCES connections(conn_alias)
+      PRIMARY KEY (host, port, snd_id),
```

Comment:
we should start making style changes as separate PRs (guilty myself of not doing it)